src/pages/index.css

```diff
@@ -26,4 +27,13 @@
 .index__newsletter {
   display: flex;
   justify-content: center;
-}
\ No newline at end of file
+}
+
+@media (max-width: 650px) {
```

Comment:
I was using another breakpoint for the mobile menu before, should we use two different ones or would this one: https://github.com/WebAudioConf/webaudioconf.github.io/blob/a5c64ed3e000c8caa06a5d3c1786310236a717a6/src/layouts/index.css#L49 work here as well?

Reply:
I had to introduce a new one because I needed the adapted style to kick in already at about 650px. And I also wanted it to work on a desktop computer which has a big screen but is showing the page in a small window.